eureka-client/src/main/java/com/netflix/appinfo/providers/Archaius1VipAddressResolver.java

```diff
@@ -0,0 +1,39 @@
+package com.netflix.appinfo.providers;
+
+import com.netflix.config.DynamicPropertyFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Archaius1VipAddressResolver implements VipAddressResolver {
```

Comment:
I don't think these two classes are really used now, only keeping them around for legacy support (and also making this change as they have config references).